### PR TITLE
XAMARIN-43498-newly added feature is updated in WinUi Scheduler

### PR DESCRIPTION
**Pull request by saiganesh-sakthivel**
XAMARIN-43498-newly added feature is updated in WinUi Scheduler is reviewed by @Chrispineagunja

### CODE REVIEW

**Review by Nijamudeen-Mohamed — APPROVED** [review]
FIne

File: winui/Scheduler/Appointment-Editing.md [hunk 1]
```diff
@@ -34,7 +34,29 @@ The Scheduler supports editing the recurrence appointment. The following editor
 
 ![edit-an-recurring-appointment-using-recurring-appointment-editor-dialog-in-winui-scheduler](Appointment-Editing_Images/edit-an-recurring-appointment-using-recurring-appointment-editor-dialog-in-winui-scheduler.png)
 
-The opening of recurrence popup editor dialog can be handled using [RecurringAppointmentEditMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.RecurringAppointmentEditMode.html) property in [AppointmentEditorOpeningEventArgs](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.AppointmentEditorOpeningEventArgs.html) by handling [AppointmentEditorOpening](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_AppointmentEditorOpening) event.
+Handle the opening of recurrence popup window using `EditMode` property in `RecurringAppointmentBeginningEditEventArgs` by handling `RecurringAppointmentBeginningEdit` event.
```
Comment (Chrispineagunja) [inline winui/Scheduler/Appointment-Editing.md:37]: Handle the opening of recurrence popup window using the `EditMode` property in the `RecurringAppointmentBeginningEditEventArgs` by handling the `RecurringAppointmentBeginningEdit` event.

File: winui/Scheduler/Appointment-Editing.md [hunk 2]
```diff
@@ -34,7 +34,29 @@ The Scheduler supports editing the recurrence appointment. The following editor
...
+
+#### RecurringAppointmentBeginningEdit Event
+
+The opening of recurrence popup editor dialog can be handled using `EditMode` property in `RecurringAppointmentBeginningEditEventArgs` by handling `RecurringAppointmentBeginningEdit` event.
```
Comment (Chrispineagunja) [inline winui/Scheduler/Appointment-Editing.md:41]: The opening of the recurrence popup editor dialog can be handled using the `EditMode` property in the `RecurringAppointmentBeginningEditEventArgs` by handling the `RecurringAppointmentBeginningEdit` event.

File: winui/Scheduler/Appointment-Editing.md [hunk 3]
```diff
@@ -34,7 +34,29 @@ The Scheduler supports editing the recurrence appointment. The following editor
...
+
+`EditMode` : Get or Sets the edit mode to perform the edit option to edit the occurrence or series for recurrence appointment. The default value of `EditMode` is `User.`
```
Comment (Chrispineagunja) [inline winui/Scheduler/Appointment-Editing.md:43]: `EditMode:` Gets or Sets the edit mode to perform the edit option to edit the occurrence or series for a recurrence appointment. The default value of the `EditMode` is `User.`

File: winui/Scheduler/Appointment-Editing.md [hunk 4]
```diff
@@ -34,7 +34,29 @@ The Scheduler supports editing the recurrence appointment. The following editor
...
+
+* User: The default editor content dialog will appear when editing a recurrence appointment to select the edit option from the end-user itself.
+* Occurrence: Edit the particular occurrence alone in a recurrence appointment. The default editor content dialog will not appear.
```
Comment (Chrispineagunja) [inline winui/Scheduler/Appointment-Editing.md:46]: * Occurrence: Edit a particular occurrence alone in a recurrence appointment. The default editor content dialog will not appear.

File: winui/Scheduler/Appointment-Editing.md [hunk 5]
```diff
@@ -34,7 +34,29 @@ The Scheduler supports editing the recurrence appointment. The following editor
...
+* Series: Edit the entire series in a recurrence appointment. The default editor content dialog will not appear.
+
+{% tabs %}
+{% highlight c# tabtitle="MainWindow.xaml.cs" hl_lines="1" %}
+this.scheduler.RecurringAppointmentBeginningEdit += scheduler_RecurringAppointmentBeginningEdit;
+
+private void scheduler_RecurringAppointmentBeginningEdit(object sender, RecurringAppointmentBeginningEditEventArgs e)
+{
+	// Get or set the edit mode to perform the edit option
```
Comment (Chrispineagunja) [inline winui/Scheduler/Appointment-Editing.md:55]: // Get or set the edit mode to perform the edit option.

File: winui/Scheduler/Appointment-Editing.md [hunk 6]
```diff
@@ -50,12 +72,6 @@ The [AppointmentEditorOpeningEventArgs](https://help.syncfusion.com/cr/winui/Syn
 
 `Cancel`: To avoid the default appointment editor showing, by enabling this property.
```
Comment (Chrispineagunja) [inline winui/Scheduler/Appointment-Editing.md:73]: `Cancel:` To avoid the default appointment editor from showing by enabling this property.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**